Source/Parallelization/WarpXComm.cpp

```diff
@@ -360,6 +368,68 @@ WarpX::UpdateAuxilaryDataSameType ()
     }
 }
 
+// TODO Add Doxygen
```

Comment:
Could you add the corresponding documentation and remove this line?

Reply:
Yes, I added the Doxygen documentation in the corresponding header file: https://github.com/ECP-WarpX/WarpX/pull/1763/commits/05dae01c43c906c3af3c273c1f50c94a1cf820c3.